Added clarity to the impacts of include gateway mode. (#6982)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-devices/warp/exclude-traffic/split-tunnels.md

2. Under **Split Tunnels**, choose a Split Tunnel mode:

    - **(default) Exclude IPs and domains** — All traffic will be sent to Cloudflare Gateway except for the IPs and domains you specify.

>     - **Include IPs and Domains** — Only traffic destined to the IP address or domains you specify will be sent to Cloudflare Gateway. **Note:** This mode will make any network or HTTP policy that is destined to a non-specified IP address ineffective as policies will _only_ be applied to the specified IPs.

Reviewer: ```suggestion
    - **Include IPs and Domains** — Only traffic destined to the IP address or domains you specify will be sent to Cloudflare Gateway. All other traffic will bypass Gateway and will no longer be filtered by your network and HTTP policies.
```
